parser.go

```diff
@@ -280,10 +303,15 @@ func (p *Parser) ParseBytes(in []byte) error {
 
 	err = p.validateFields()
 	if err != nil {
-		for _, err := range err.(ValidationErrors) {
-			err.Line, err.Column = getPositionInFile(err.Key, node)
-
-			ve = append(ve, err)
+		for _, err := range err.(ValidationResults) {
```

Comment:
I'm not really sure if using a type for a slice of `error`s is the kosher thing to do in Go, but the parser did it before with `ValidationErrors` :shrug:

Reply:
No it's, ok, i've already seen that when dealing with external's APIs error responses that come from an SDK.